node/pkg/reporter/types.go

```diff
@@ -123,23 +112,28 @@ func WithLatestData(latestData *sync.Map) ReporterOption {
 	}
 }
 
+func WithLatestSubmittedData(latestSubmittedData *sync.Map) ReporterOption {
+	return func(c *ReporterConfig) {
+		c.LatestSubmittedData = latestSubmittedData
+	}
+}
+
 type Reporter struct {
 	KaiaHelper         *helper.ChainHelper
-	SubmissionPairs    map[int32]SubmissionPair
+	Pairs              []string
 	SubmissionInterval time.Duration
 	CachedWhitelist    []common.Address
 
 	contractAddress    string
 	deviationThreshold float64
 
-	LatestData *sync.Map // map[symbol]SubmissionData
-	Job        func() error
+	LatestData          *sync.Map
```

Comment:
I'd like to suggest naming updates
`LatestDataMap`
and
`LatestSubmittedDataMap`

it seems a bit confusing that getLatestData function is getting latestData as parameter, so that's why i'm suggesting

Reply:
oh it'd go same for opt and reporterConfig

Reply:
done!

Reply:
sorry can you also update the opt function name?
ex) WithLatestDataMap

Reply:
done as well! 😃